src/css/main.scss

```diff
@@ -494,6 +494,8 @@ tr:last-of-type {
 }
 
 .color-img {
+  font-size: 0;
```

Comment:
This is to ensure that an img inside a div takes up the full height of the div.

Reply:
did you add CSS reset? that might help for this

Reply:
Yes! I tried adding a rule for divs to have `font-size: 0, line-height: 0`, but this affects all divs so is it ok to keep this in the `color-img` class? (all images follow this format: `<div class="color-img"><img src=""/></div>`)